distributed/protocol/core.py

```diff
@@ -122,7 +122,23 @@ def loads(frames, deserialize=True, deserializers=None):
             else:
                 value = Serialized(head, fs)
 
-            get_in(key[:-1], msg)[key[-1]] = value
+            from toolz import get_in
+
+            def put_in(keys, coll, val):
```

Comment:
Explicitly upcasts tuples as needed into lists here.

Reply:
Since this is in `distributed.protocol.core.loads`, my assumption is that if we use choose to `loads_msgpack(use_lists=True)` here i guess all our work might be for naught unless we take this approach